wrapper.sh

```diff
@@ -328,28 +437,54 @@ if [[ $import == true ]]; then
    #add update lessons to remote
    cd ../${repo}-ja
    git init
-   remotes=`git remote | grep "swc" | wc -l`
+   remotes=`git remote | grep "swc-ja" | wc -l`
    if [[ remotes -le 0 ]]; then
-       git remote add swc https://github.com/$git_user/$repo-ja.git 
+       git remote add swc-ja https://github.com/$remote_user/$repo-ja.git 
     fi
-   git pull swc master
+   git pull swc-ja master
+
+   # remove files provided by template
+   rm -rf bin/boilerplate
+   rm -rf _layouts _includes _episodes_rmd assets bin code 
+
    git add *
    git commit -m "update lesson files"
-   git push swc master
+   git push swc-ja master
 
    #update original lesson to import translated content
    cd ../i18n # or English lesson
    git submodule foreach git pull origin master
 
-#####################################
-# to do : create locale submodule   #
-# in original lesson if none exists #
-###############$#####################
-   
+  #restore locale lessons (only English lessons translated)
+  for dir in `git submodule |  grep "^+"  | cut -d" " -f2`
+    do
+    if [ -d $dir ] 
+      then
+      cd $dir
+      git reset --hard  
+      cd ..
+    fi
+  done  
+
   cd ${repo}
   git add -u
+
+  #create lesson in locale if not existing or update
+  if [ -d ./locale/ja ]
+    then
+    cd locale/ja
+    git pull origin master
+    cd ../..
+  else
+    mkdir -p locale
+    url="https://github.com/$remote_user/${repo}-ja.git"
+    git submodule add $url ./locale/ja
+  fi
+
+  #push updated locale lessons to English lesson
+  git add -u
```

Comment:
This commits updates to the submodules (adding new lessons in `repo/_locale/ja`)